src/components/about/About.js

```diff
@@ -49,23 +48,23 @@ const About = () => {
             <img className="icons" src={categorizeIcon} alt="" />
             <h2>Categorize</h2>
             <p>
-              We have generated a model with in our data base to catergorize
-              each incident report we recive. Because our system is in beta,
-              there may be some false positives (e.g.,incident reports where law
-              enforcement were present but no use-of-force incidents actually
-              occurred), and some false negatives (e.g., incident reports where
-              use-of-force occurred and were reported but do not make it into
-              our system). To combat this, each incident report needs to be
-              approved by the Blue Witness Administrative Team.
+              We generated a model within our database to categorize each
```

Comment:
Suggesting we change to:
"Our BERT model categorizes each incident report we receive."